spec/features/lease_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.feature 'leases' do
+  let(:user) { create(:user) }
+  before do
+    sign_in user
+  end
+  describe 'create a new leased object' do
+    let(:future_date) { 5.days.from_now }
+    let(:later_future_date) { 10.days.from_now }
+
+    # Pending revision to work in hyrax
+    # TODO: https://github.com/samvera-labs/hyrax/issues/1123
```

Comment:
OK pushed up this change in a WIP commit to see if it works out.  However, the feature specs may be entirely revised after the design work is done to integrate these features into hyrax.  If this spec-metadata addition does not work, I suggest the feature specs are re-written once the design work is done, unless there is a clear expectation that all these CC views _should_ be simply plug-n-play in hyrax (of course, this was not explicit in the requirements for this work).